<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>604</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/按钮图标/窗体/res/图标.png</normaloff>:/按钮图标/窗体/res/图标.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基础设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>模式</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>普通模式</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>该模式较为稳定，参数适中，适合日常使用</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>该模式进一步提高操作速度，取消所有操作时间间隔</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="PlaceholderText">
                <brush brushstyle="SolidPattern">
                 <color alpha="128">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="PlaceholderText">
                <brush brushstyle="SolidPattern">
                 <color alpha="128">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>120</red>
                  <green>120</green>
                  <blue>120</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>120</red>
                  <green>120</green>
                  <blue>120</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="PlaceholderText">
                <brush brushstyle="SolidPattern">
                 <color alpha="128">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>极速模式</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>图形点击设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1" colspan="3">
             <widget class="QSlider" name="horizontalSlider_2">
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="pageStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="3">
             <widget class="QSlider" name="horizontalSlider_4">
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="pageStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>精度</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>毫秒</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>毫秒</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="3">
             <widget class="QSlider" name="horizontalSlider_3">
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="pageStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>持续时间：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>时间间隔：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>暂停时间：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="4">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>毫秒</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="5">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>指每完成一次指令至下一次指令开始的时间间隔</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="5">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>指执行指令如鼠标点击的持续时间</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="5">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>指鼠标双击的时间间隔</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>图像匹配精度：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QSlider" name="horizontalSlider">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>9</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其他设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>百度OCR设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>secretKey：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>appId：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>apiKey：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>前往网页</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>253</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>每次退出提示清空指令</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>恢复至默认</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>每次启动检查更新</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>系统提示音</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>任务完成后显示主窗口</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
